editor/quick_open.cpp

```diff
@@ -118,6 +118,11 @@ void EditorQuickOpen::_update_search() {
 float EditorQuickOpen::_score_path(const String &p_search, const String &p_path) {
 	float score = 0.9f + .1f * (p_search.length() / (float)p_path.length());
 
+	// Exact match.
+	if (p_search == p_path) {
+		return 1.2f;
```

Comment:
Is that `1.2` a magic number somehow? There should be a comment to describe why that particular value is returned.

Reply:
I have no idea why it's special, I just compared the quick open scene of 3.x and 4.0 and found that it was missing this if statement. All I knew was that things worked as they should in 3.x, so this seems to have replicated that.